cn/ios/objects.mdown

```diff
@@ -293,8 +578,14 @@ shield.rupees = 50;
 
 1.  声明符合 `PFSubclassing` 协议的子类。
 2.  执行子类方法 `parseClassName`。您可以将该字符串传送至 `initWithClassName:`，这样后续就不必进行类名引用。
-3.  将 `PFObject+Subclass` 导入您的 .m 文件。该操作可以实施 `parseClassName` 以外的 `PFSubclassing` 中的所有方法。
-4.  在您的 ApplicationDelegate 中调用 `[YourClass registerSubclass]`，然后调用 Parse `setApplicationId:clientKey:`。
+3.  将 `PFObject+Subclass` 导入您的 .m 文件。该类实现了 `parseClassName` 以外的 `PFSubclassing` 中的所有方法。
+4.  在ApplicationDelegate 中调用 Parse `setApplicationId:clientKey:`之前调用 `[YourClass registerSubclass]`。
+
+
+一个更简单的方法是使用 '[+load](https://developer.apple.com/library/ios/documentation/cocoa/reference/foundation/classes/nsobject_class/reference/reference.html#//apple_ref/occ/clm/NSObject/load) (Obj-C有效) 或者 [initialize](https://developer.apple.com/library/ios/documentation/Cocoa/Reference/Foundation/Classes/NSObject_Class/#//apple_ref/occ/clm/NSObject/initialize) （Obj-C和Swift都有效）方法来实现子类化。
```

Comment:
In en/iOS/objects.markdown, it said  "Call `[YourClass registerSubclass]` before Parse `setApplicationId:clientKey:`
".Is means that call the `setApplicationId:clientKey:' first?